inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/oracle-cdc/src/main/java/org/apache/inlong/sort/cdc/oracle/source/config/OracleSourceConfigFactory.java

```diff
@@ -108,10 +108,6 @@ public OracleSourceConfig create(int subtaskId) {
             props.setProperty("database.port", String.valueOf(port));
         }
 
-        if (schemaList != null) {
```

Comment:
Why change the code of Oracle cdc?

Reply:
schemaList is never used for pg  and oracle